.github/workflows/ci.yml

```
@@ -316,3 +316,38 @@ jobs:
         with:
           github_token: ${{ github.token }}
           filter_mode: file
+
+  check-notebook-format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install requirements
+        run: |
+          pip install nbconvert
+          pip install nbformat
+          pip install black
```

I think this can be one line, and, more importantly, I think the black version should be fixed, so that a black upgrade doesn't randomly change our formatting (different to our real dependencies, because this is just an internal tool).

```suggestion
          pip install nbconvert nbformat black==19.10b0
```

The one-line-ing could be applied to the checker too.

Finally, the GitHub Actions Python guide suggests upgrading pip as the first step: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#installing-dependencies